src/taco-cli/cli/utils/templateManager.ts

```diff
@@ -236,40 +186,32 @@ class TemplateManager {
             });
     }
 
-    private acquireTemplate(templateId: string, kitId: string): Q.Promise<string> {
+    private acquireTemplate(templateId: string, kitId: string, tempTemplateDir: string = os.tmpdir()): Q.Promise<string> {
```

Comment:
+1. If there are no scenarios where you need to configure tempTemplateDir, you can avoid passing it as a param to acquireTemplate, acquireFromGit and acquireFromTacoKits methods and use os.tmpdir() instead.